docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -49,22 +49,9 @@ The `sum_anosmia_ageusia_raw_search` signals are simply the raw sum of the
  than one symptom. Currently, Google does not provide _intersection/union_
  data. Users should be careful when considering such signals.
 
-## Limitation 
-When daily volume in a region does not meet quality or privacy thresholds, set
-by Google, no value will be reported. Since Google uses differential privacy,
-there is artificial noise added to the raw datasets to avoid identifying any
-individual persons without affecting the quality of results.
-
-The data is normalized by the total number of Search users in certain regions
-for a certain time period and is scaled considering the maximum value of the
-normalized popularity across the entire published time range for that region
-over all symptoms. The values of symptom popularity are **NOT** comparable
-across geographic regions. Due to the scaling step, most of the values should be
-in the range 0-1. However, since the scaling factor is calculated and stored at
-a certain time point, the symptom popularity released after that time point is
-likely to exceed the previously-observed maximum value which results in values
-larger than 1.
-
+## Properties and limitations 
+To learn more about the dataset, how it is generated it and its limitations, 
+read the [dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```

Comment:
It's good to link out for more details, but I don't like the idea of hiding the region non-comparability limitation in page 3 of an external link. We can put a summary here for responsible caveat emptor purposes and let people who need the specifics click out.

```suggestion
When daily volume in a region does not meet quality or privacy thresholds, set
by Google, no value will be reported.

Google uses differential privacy, which adds artificial noise to the raw
datasets to avoid identifying any individual persons without affecting the
quality of results.

Google normalizes and scales time series values to determine the relative
popularity of symptoms in searches within each geographical region individually.
This means that the resulting values of symptom popularity are **NOT**
comparable across geographic regions. 

More details about the limitations of this dataset are available in [Google's Search 
Trends symptoms dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```

Reply:
I'm ok with this proposal and most of the text, but notice that when daily data is not available we are offering weekly data so perhaps it can be adjusted to be either:
"When daily volume in a region does not meet quality or privacy thresholds, set by Google, no daily value will be reported."
Or better:
"When daily volume in a region does not meet quality or privacy thresholds, set by Google, data will be provided at the weekly resolution. If the quality or privacy thresholds are not met at the weekly resolution, no data will be reported for the symptom in that region."

Reply:
Yeah, but it won't show up in the COVIDcast data even if it's available in Google's weekly dataset. How about:

```suggestion
When daily volume in a region does not meet quality or privacy thresholds, set
by Google, no daily value is reported. Weekly data may be available from Google 
in these cases, but we do not yet support importation using weekly data.

Google uses differential privacy, which adds artificial noise to the raw
datasets to avoid identifying any individual persons without affecting the
quality of results.

Google normalizes and scales time series values to determine the relative
popularity of symptoms in searches within each geographical region individually.
This means that the resulting values of symptom popularity are **NOT**
comparable across geographic regions. 
More details about the limitations of this dataset are available in [Google's Search 
Trends symptoms dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```

Reply:
Sounds good. 
Please hyperlink the words "available from Google" to https://pair-code.github.io/covid19_symptom_dataset.